python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -251,7 +251,11 @@ def op_def(self) -> OpDefinition:
     @public
     @property
     def has_partition_key(self) -> bool:
-        """Whether the current run is a partitioned run."""
+        """Whether the current run is a partitioned run.
```

Comment:
This is a preexisting thing, but may as well change it while we're here--can we clarify here that `has_partition_key` only returns true for single partition runs (and not runs that cover a range of partitions)?

This was reported by a user: https://dagster.slack.com/archives/C01U5LFUZJS/p1692726135743139